static/docs/user-guide/large-dataset-optimization.md

```diff
@@ -120,6 +120,13 @@ file link types. Please refer to the
 [Update a Tracked File](/doc/user-guide/updating-tracked-files) on how to manage
 tracked files under these cache configurations.
 
+### Re-linking data in the workspace
+
+To re-create the file links in the workspace, for example after changing the
```

Comment:
re-create ----> make sure that link are of the right type and recreate if needed?

Reply:
I don't know if the "if needed" part is accurate. Waiting for Alex' reply in https://github.com/iterative/dvc.org/pull/864#pullrequestreview-338379551 above. But I simplified this sentence along the lines you suggested. Thanks.